### ISSUE TITLE
Crash when loading MP4/GIF into default Cleanup or Schematic room

### ISSUE THREAD

**Issue by RattusaurusRex** [opentoonz/opentoonz#4384]
Attempting to load an MP4 or GIF file into Opentoonz while in either the default Cleanup or Schematic room results in a crash.  Other rooms do not result in a crash.  I tested this in a fresh install of Opentoonz with default settings.

**To Reproduce**
1.  Go to the Cleanup or Schematic room
2.  Load an MP4 or GIF file either by dragging it into the program or through Load Level
3.  Opentoonz hangs, some panels go blank, and crashes

**Expected behavior**
Loading in any room without crashing

**Screenshots**
![image](https://user-images.githubusercontent.com/60277069/160992052-1f418bfe-5f36-4da1-8fd6-28e2208aaa78.png)

https://user-images.githubusercontent.com/60277069/160992205-1914bef2-abab-40f6-9524-94132633281e.mp4

Device Information:

    OS: Windows 10 Pro 64-bit
    OpenToonz Version: 1.6 Nightly build 2022-03-31 05:36:38 UTC
    CPU: AMD Ryzen Threadripper 2990WX 32-Core Processor
    GPU: NVIDIA GeForce RTX 3070 Ti
    Graphics Tablet: Wacom Cintiq Pro 24

**Comment by shun-iwasawa** [comment]
Unfortunately I couldn't reproduce the crash so far..

**Comment by RodneyBaker** [comment]
I spent quite a bit of time with @RattusaurusRex on discord even getting a copy of his Opentoonz installation as a portable and could not replicate the problem.

Two things I note is that the problematic system is:  1) Not entirely a clean install* 2) Highly customized

*Files in the Program/Opentoonz directory include older support files such as Opencvworld310.dll which I've noticed on other people's computers.  I'm not suggesting that relates to the issue but it does suggest this is not a completely clean install.

**Comment by RattusaurusRex** [comment]
@RodneyBaker No, it still crashes if the Opentoonz program folder is deleted before install.

**Comment by RodneyBaker** [comment]
Understood.  I was just describing what I personally had tested.
Both your stuff directory and program directory were not default installation as each had elements of additional files and customizations.  You may recall the customized rooms and the older opencv dll file present in what we tested.

If you have not already done so I'll recommend renaming your stuff directory so the installer will provide all new and default files in that directory to test.  If the Opentoonz program folder was deleted I would hope that deletion was only after running the uninstaller as the uninstaller is required to remove entries in the Windows registry.  The manual deletion of the directory thereafter is then to remove any files not removed by the uninstaller left over from other installations.

**Comment by RattusaurusRex** [comment]
@RodneyBaker I already told you that it was a fresh install, otherwise it would not give me the default rooms.  No Opentoonz folder, no Opentoonz stuff folder.  It crashes.

**Comment by artisteacher** [comment]
Does the version of FFMPEG make a difference? I think one of the PR's this winter recommended updating to the most recent version of FFMPEG.

**Comment by RattusaurusRex** [comment]
@artisteacher I thought maybe that too, so I updated my ffmpeg and it still crashed.

**Comment by RattusaurusRex** [comment]
The images are getting loaded into the ffmpeg cache folder before crashing:

https://user-images.githubusercontent.com/60277069/161164031-5e54c99c-0fc7-44b7-aed2-93d6e6024dca.mp4

**Comment by shun-iwasawa** [comment]
Sorry but this issue cannot be resolved before V1.6 release since I'm still failing to reproduce the crash . 
Please use rooms other than Cleanup or Schematic for loading mp4 as a temporal workaround.

**Comment by RattusaurusRex** [comment]
I'll add that this crash affects custom rooms, not just default rooms.  So 6 of my own rooms crash: 

![image](https://user-images.githubusercontent.com/60277069/161197083-161ec04f-ea2c-4efc-8f33-9347d9fd4f69.png)


I've also figured out when the crash started.  I have no issues with loading until this nightly: https://github.com/opentoonz/opentoonz_nightlies/releases/tag/nightly_424c2637b02a73808570cb54b1602b031f00df1c

**Comment by justburner** [comment]
#4205 do patch the FFMpeg waiting to still refresh OpenToonz, it was mainly for export but it also get applied on import, this was done so the application don't get frozen and completely unresponsive each time it interacts with FFMpeg.

It does by calling Qt events, that also means any widget can be refreshed while FFMpeg is still working, if the widget don't sanity check if the actual frame exists it can be troublesome.

In another words, the real crash can be anything, the patch just opened the doors for those existing bugs.

Without the patch, OT remains completely frozen until all frames are fully imported... if the sequence is big then it can remain frozen for a loooooong time and OS can even suggest you to kill the unresponsive app.

**Comment by artisteacher** [comment]
It's interesting that the schematic room crashes but not the animation room - they have all the same windows except that the animation room also has a level strip and xsheet. 

I'm not surprised by cleanup - it's historically been very crash happy on macOS.

**Comment by RattusaurusRex** [comment]
I also can't load a scene that contains an mp4 or gif without crashing in certain rooms, including my Loading room now (which makes that room pretty useless if I can't load files in it). 

First clip is using 2021-12-29 nightly, second is 2022-01-05 nightly, third is 2022-04-27 nightly.
https://user-images.githubusercontent.com/60277069/167711889-e83706cf-64e1-49fd-aea2-051bbf2f035f.mp4
